### PR TITLE
PR: Fix some hover and calltip issues

### LINKED ISSUE #9311: Some issues with calltips and hovers
@goanpeca, I noted the following issues the last couple of days:

* `Press Ctrl+I for additional help` is misleading right now because calltips are shown after writing a left parenthesis, and pressing `Ctrl+I` does nothing at that point (as you already noticed). So please remove that message. We'll restore it once you solve issue #9279.

* There's no documentation shown for calltips while it's shown for hovers. Please take a look at what's happening with that. Screenshots:

    Selección_005

    Selección_006

* The results of pressing `Ctrl+I` next to an object and pressing a hover to get documentation are different. Screenshots attached:

    - `Ctrl+I`
    Selección_003

    - Pressing on hover
    Selección_004

* The placement of hovers is covering the object name a little bit, so I think they should be placed a little bit below. For example, you can't  correctly read `vectorize` in the screenshot below

    Selección_005

### PR DESCRIPTION
**Pull request by goanpeca**
## Description of Changes

hints2

### Issue(s) Resolved

Fixes #9311

### Affirmation

By submitting this Pull Request or typing my (user)name below,
I affirm the [Developer Certificate of Origin](https://developercertificate.org)
with respect to all commits and content included in this PR,
and understand I am releasing the same under Spyder's MIT (Expat) license.

I certify the above statement is true and correct: @goanpeca

### CODE REVIEW

**Comment by goanpeca** [comment]
@ccordoba12 I believe this fixes all the issues you presented on #9322. Let me know if it works as expected

File: spyder/widgets/calltip.py [hunk 1]
```diff
@@ -39,6 +39,7 @@ class ToolTipWidget(QLabel):
     """
 
     sig_help_requested = Signal(str)
+    sig_closed = Signal()
```
Comment (ccordoba12) [inline spyder/widgets/calltip.py]: This signal is not used in this PR. If it's not necessary, please remove it.

**Comment by ccordoba12** [comment]
There's another important issue you need to take care of: hovers are not placed correctly if the line you're trying to inspect is close to the bottom of the screen (but calltips are).

Selección_007

Selección_008

**Comment by ccordoba12** [comment]
And another thing I just noticed in the screenshots above: the signature of `arange` is repeated in its calltip but not in its hover.

**Comment by goanpeca** [comment]
Thanks, will fix it!

**Comment by goanpeca** [comment]
@ccordoba12 fixed the issues. Another tests is failing on macos and py2.7 but locally is passing, so I am not sure what the issue might be 🤷‍♂ 

hints3

**Pull request merged**